webpack.config.js

```diff
@@ -88,4 +88,11 @@ function buildConfig(envConfig) {
 
 const envConfig = getConfig();
 
-module.exports = buildConfig(envConfig);
+module.exports = {
+  externals: {
+    url: {
+      commonjs: "URL"
+    }
+  },
+  webpackConfig: buildConfig(envConfig)
+};
```

Comment:
this seems weird. wouldn't we want to do externals in `buildConfig`

Reply:
Yes, it needs to be moved there, per my gist